vertical-pod-autoscaler/pkg/updater/logic/updater.go

```diff
@@ -85,6 +105,19 @@ func (u *updater) RunOnce(ctx context.Context) {
 	timer := metrics_updater.NewExecutionTimer()
 	defer timer.ObserveTotal()
 
+	if u.useAdmissionControllerStatus {
+		isValid, err := u.statusValidator.IsStatusValid(status.AdmissionControllerStatusTimeout)
+		if err != nil {
+			klog.Errorf("Error getting Admission Controller status: %v. Skipping eviction loop", err)
+			return
+		}
+		if !isValid {
+			klog.Warningf("Admission Controller status has not been updated for %v. Skipping eviction loop",
```

Comment:
nit: Admission Controller status has been refreshed more than %v ago.
Also, how are Duration object printed? Will the log be readable?

Reply:
Done.
Duration has String() method implemented. When %v is called it will print result of String() method if available. So it will be readable.